chat-commands.js

```diff
@@ -3491,7 +3560,7 @@ exports.commands = {
 		room.battle.endType = 'forced';
 		if (!target) {
 			room.battle.tie();
-			this.logModCommand(user.name + " forced a tie.");
+			this.modlog("FORCETIE");
```

Comment:
You're using ``'`` marks consistently for ``this.modlog`` up until here, why change?

Reply:
my bad coding late at night